app/controllers/orders/detail.js

```diff
@@ -159,6 +181,23 @@ export default GoodcityController.extend({
   },
 
   actions: {
+    cancelOrder() {
+      const params = {
```

Comment:
It'd be nice to let the service build the params object itself, the controller doesn't really need to.
Definitely not a big deal, just mentioning it